demo/components/presentational/SellOrder.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { StyleSheet, View, Text } from "react-native";
+import PropTypes from "prop-types";
+import { responsiveWidth } from "react-native-responsive-dimensions";
+import EstateForSale from "./EstateForSale";
+
+export default function SellOrder({ sellOrder }) {
+  const { asset: estate, priceMana } = sellOrder;
+
+  // Note: Conversion to USD will be implemented on v0.2.0
+  // <Text>{priceMana} MANA (~${sellOrder.priceUsd})</Text>
+  return (
+    <View style={styles.landContainer}>
+      <EstateForSale estate={estate} />
+      <Text>{priceMana} MANA</Text>
```

Comment:
I wonder whether we want to style the number and the word mana with different fonts, fontsizes, fontweights, etc.

Reply:
Yes, probably. Sorry but styling isn't mt strength...

Reply:
I'm couldn't think on any suggestion here.

Reply:
Okay! I'll split this out into a separate issue.

Reply:
https://github.com/tasitlabs/tasit/issues/87

Resolving here